environs/simplestreams/datasource_test.go

```diff
@@ -4,12 +4,12 @@
 package simplestreams_test
 
 import (
+	"github.com/juju/utils"
```

Comment:
This is in the wrong stanza, but we can follow this up as a driveby later.

Reply:
I had fixed this in develop, but forgot to back port that small pr as well.  Now it's been wiped out by a merge from 2.7 to develop.  :-(